src/core/qgsproject.h

```diff
@@ -991,6 +1010,8 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
 
     std::unique_ptr<QgsMapThemeCollection> mMapThemeCollection;
 
+    std::unique_ptr<QgsLabelingEngineSettings> mLabelingEngineSettings;
```

Comment:
What's the reason for storing this in a unique_ptr? Wouldn't storing this by value work the same way with (a tiny bit) less overhead?